src/components/core/tests/PivotTable.spec.tsx

```diff
@@ -69,6 +69,12 @@ describe("PivotTable", () => {
         expect(wrapper.find(PivotTableInner)).toHaveLength(1);
     });
 
+    it("should call passed loading component", () => {
+        const CustomLoadingComponent = jest.fn().mockReturnValue(<div />);
+        renderComponent({ LoadingComponent: CustomLoadingComponent });
+        expect(CustomLoadingComponent).toHaveBeenCalled();
+    });
+
```

Comment:
There should also be an opposite test that test usage of internal LoadingComponent when prop is undefined/null (the default props). I don't see it in the file. The original author probably forgot about it. Can you add it, please?

Reply:
I also rewrote the former one to correspond with the latter.